docs/setup_docs.sh

```diff
@@ -35,9 +35,23 @@ echo "Created temporary file" $goModFileLocation/go.mod
 # Make Hugo retrieve modules which are used for externally hosted documentation
 currentBranch=$(git rev-parse --abbrev-ref HEAD)
 
+function integrate_connector_docs {
+  connector=$1
+  ref=$2
+  git clone --single-branch --branch ${ref} https://github.com/apache/flink-connector-${connector}
+  theme_dir="../themes/connectors"
+  mkdir -p "${theme_dir}"
+  rsync -a flink-connector-${connector}/docs/content* "${theme_dir}/"
+}
+
 if [[ ! "$currentBranch" =~ ^release- ]] || [[ -z "$currentBranch" ]]; then
-  # If the current branch is master or not provided, get the documentation from the main branch
```

Comment:
Should we leave the comments here to clarify why we're doing this?

Reply:
My plan was to just remove these conditions.

AFAICT we don't need them; each release will have an explicit list of integrated connector docs and versions.